apps/condo/domains/ticket/schema/TicketHintProperty.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Generated by `createschema ticket.TicketHintProperty 'organization:Relationship:Organization:CASCADE;ticketHint:Relationship:TicketHint:CASCADE; property:Relationship:Property:SET_NULL;'`
+ */
+
+const { Relationship } = require('@keystonejs/fields')
+
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+const access = require('@condo/domains/ticket/access/TicketHintProperty')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+
+const TicketHintProperty = new GQLListSchema('TicketHintProperty', {
```

Comment:
Is it not a redundant schema in your case? AFAIK, Keystone can generate a schema `Ticket_properties_many` if you specify `many: true` for a `TicketHint`.

Reply:
No, it's better to create an explicit model than to use `many: true`. `many: true` results in very expensive queries, we want to move away from this in the future.